digitalpy/core/persistence/persistent_object.py

```diff
@@ -2,7 +2,7 @@
 from typing import Any
 
 from uuid import uuid4
-from digitalpy.model.object_id import ObjectId
+from digitalpy.core.domain.object_id import ObjectId
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0